src/main/java/io/opentracing/BraveSpanBuilder.java

```diff
@@ -138,4 +139,20 @@ BraveSpanBuilder withStateItem(String key, Object value) {
         }
         return this;
     }
+
+    @Override
+    public long getContextTraceId() {
+        return traceId;
+    }
+
+    @Override
+    public long getContextSpanId() {
+        return parentSpanId;
+    }
+
+    /** null **/
+    @Override
+    public Long getContextParentSpanId() {
+        return null;
```

Comment:
Is it ok to always return null here? (instead of grabbing the parentParentId as traceId/parentSpanId do)